cmd2/cmd2.py

```diff
@@ -3469,15 +3469,18 @@ def _initialize_history(self, hist_file):
 
         # first we try and unpickle the history file
         history = History()
+        # on Windows, trying to open a directory throws a permission
+        # error, not a `IsADirectoryError`. So we'll check it ourselves.
+        if os.path.isdir(hist_file):
```

Comment:
This seems like a reasonable solution.

Reply:
Pathlib on python 3.4 doesn't have `expanduser()`. Switching back to `os.path`

Reply:
Ah good call.  I thought there was a reason I was waiting until we deprecated Python 3.4 before I started adding `pathlib` usage, but I couldn't recall what it was.  Thats it!